main.go

```diff
@@ -2,8 +2,14 @@ package main
 
```

Comment:
The extra functions at the bottom of this file should be in config.go or sshcode.go, main.go is just for parsing args and running the sshcode function.